media/com_joomlaupdate/js/admin-update-default.es6.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
+
+
+document.addEventListener('DOMContentLoaded', () => {
+  window.joomlaupdate_password = Joomla.getOptions('joomlaupdate_password');
```

Comment:
Once the above has been done, change this to:

```
const JoomlasUpdateOptions = Joomla.getOptions('joomlaupdate');
window.joomlaupdate_password = JoomlasUpdateOptions.password;
window.joomlaupdate_totalsize = JoomlasUpdateOptions.totalsize;
window.joomlaupdate_ajax_url = JoomlasUpdateOptions.ajax_url;
window.joomlaupdate_return_url = JoomlasUpdateOptions.return_ur;
window
```

Reply:
Done.